src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
**Hint** You only need one Lottery class instance and then You can get new balls by calling a 'getRandomBall()' method on that object. This can be done in a for loop to avoid duplication